services/notifications/README.md

```diff
@@ -2,3 +2,66 @@
 
 The notification service is responsible for sending emails to users informing them about events that happened. To do this it hooks into the event system and listens for certain events that the users need to be informed about.
 
+## Email Notifications
```

Comment:
```suggestion
## Email Notification Templates
```
I know, we just have changed that, but I would like to further improve to make it more clear 😅